lib/devise-security/validators/password_complexity_validator.rb

```diff
@@ -0,0 +1,29 @@
+module DeviseSecurity
```

Comment:
This file has a bunch of Rubocop warnings.

Reply:
Fixed.